app/templates/partials/questions/repeatinganswer.html

```diff
@@ -20,7 +20,7 @@ <h3 class="neptune">{{ _('Person ') }}<span class="js-household-loopindex">{{ lo
           {%- if loop.index == 1 -%}
              <span class="js-household-action">{{ _('You') }}</span>
           {%- else -%}
-            <button class="btn btn--link pluto js-btn-remove" name="action[remove_answer]" value="{{ loop.index0 }}" type="button">{{ _('Remove') }}
+            <button class="btn btn--link pluto js-btn-remove" name="action[remove_answer]" value="{{ loop.index0 }}" type="submit">{{ _('Remove') }}
```

Comment:
@samiwel @jonnyshaw89 I believe the non-js version should stick to the browser defaults (ie. the first button with `type=submit` submits the form). with JS, we remove that attribute and handle those actions only on click—meaning the browser will do exactly as the DOM describes.